controllers/commentController.js

```diff
@@ -142,13 +137,182 @@ export default class CommentController {
         ]
       });
       if (!comment) {
-        res
-          .status(BAD_REQUEST)
+        return res
+          .status(NOT_FOUND)
           .json({ message: "There is no comment with that id" });
       }
       res.status(OK).json({ comment });
     } catch (error) {
-      res.status(INTERNAL_SERVER_ERROR).json({ message: "Sorry, this is not working properly. We now know about this mistake and are working to fix it" });
+      res.status(INTERNAL_SERVER_ERROR).json({
+        message:
+          "Sorry, this is not working properly. We now know about this mistake and are working to fix it"
+      });
+    }
+  }
+
+  /**
+   * @description Allows users to fetch a particular comment of an article.
+   * @param  {Object} req - The request object.
+   * @param  {Object} res - The response object.
+   * @returns {Object} - It returns the request response object.
+   */
+  static async fetch(req, res) {
+    try {
+      const { id: commentId } = req.params;
+      const comment = await Comment.findOne({
+        where: { id: commentId },
+        include: [
+          {
+            model: User,
+            as: "likes",
+            attributes: ["username"],
+            through: { attributes: [] }
+          },
+
+          { model: User, as: "author", attributes: ["username", "image"] }
+        ]
+      });
+
+      if (!comment) {
+        return res.status(NOT_FOUND).json({ message: "The comment does not exist" });
+      }
+      res.status(OK).json({ comment });
+    } catch (error) {
+      res.status(INTERNAL_SERVER_ERROR).json({ message: error.message });
+    }
+  }
+
+  /**
+   * @description Allows users to fetch all comments of an article.
+   * @param  {Object} req - The request object.
+   * @param  {Object} res - The response object.
+   * @returns {Object} - It returns the request response object.
+   */
+  static async fetchAllComments(req, res) {
+    try {
+      const { slug } = req.params;
+
+      const article = await Article.findOne({
+        where: { slug },
+        include: [
+          {
+            model: Comment,
+            attributes: ["body", "id", "createdAt", "updatedAt", "like"],
+            include: [
```

Comment:
@YvesIraguha You don't need to return likes' object when fetching comments. Comments can only have likes' count instead of the full object and likes should have retrieved and paginated separately.